minichain/mempool.py

```
@@ -1,20 +1,55 @@
 import logging
 import threading
+from typing import TYPE_CHECKING, Callable, Optional
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)